pkg/runtime/runtime-manifest.go

```diff
@@ -100,6 +102,33 @@ func resolveCredential(cd manifest.CredentialDefinition) (string, error) {
 	}
 }
 
+// resolveOutput will attempt to resolve an output value if required by string interpolation
+// e.g. via {{ bundle.outputs.foo }}
+// It looks up the claim for the value written by the last action and, if exists, returns this value
+func (m *RuntimeManifest) resolveOutput(output manifest.OutputDefinition) (string, error) {
+	if m.Action != manifest.ActionInstall {
```

Comment:
I believe we have upcoming work to support running stateless custom actions before install is executes, i.e. without a claim. I suggest changing this to check if the claim is present, and if not, skip without relying on the action.

Reply:
Good call.  Will update to check if the claim is present, as suggested.